activestorage/dummy_data.py

```diff
@@ -79,6 +87,10 @@ def make_ncdata(filename, chunksize, n, compression=None,
 
     For the purposes of test data, bounds (valid_min, range etc)
     need to be non-zero, although that wont hold in real life.
+
+    pathological = True makes half the data missing so we can 
+    ensure we find some chunks which are all missing ... Can 
+    only be used in combination with a missing value.
     """
```

Comment:
we need a safety condition for that eg `if missing_data and not missing_value: toss the sink out the window`

Reply:
agreed